<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConf</class>
 <widget class="QDialog" name="DialogConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelConfIp">
     <property name="text">
      <string>IP ddr:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditIp1">
     <property name="text">
      <string>127</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelPoint">
     <property name="text">
      <string> .</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineEditIp2">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="labelPoint2">
     <property name="text">
      <string> .</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="lineEditIp3">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="labelPoint3">
     <property name="text">
      <string> .</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="lineEditIp4">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelConfPort">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelPwd">
     <property name="text">
      <string>Pwd:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="7">
    <widget class="QLineEdit" name="lineEditPwd">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="labelPwdConf">
     <property name="text">
      <string>Pwd Conf:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="7">
    <widget class="QLineEdit" name="lineEditPwdConf">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="2">
    <widget class="QRadioButton" name="radioServer">
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBoxConf">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>8866</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxConf</sender>
   <signal>accepted()</signal>
   <receiver>DialogConf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxConf</sender>
   <signal>rejected()</signal>
   <receiver>DialogConf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
